src/main/java/org/mockito/internal/creation/instance/ConstructorInstantiator.java

```diff
@@ -107,10 +107,7 @@ private String constructorArgsString() {
     private InstantiationException multipleMatchingConstructors(Class<?> cls, List<Constructor<?>> constructors) {
         return new InstantiationException(join("Unable to create instance of '" + cls.getSimpleName() + "'.",
                 "Multiple constructors could be matched to arguments of types " + constructorArgTypes() + ":",
-                join("", " - ", constructors),
-                "If you believe that Mockito could do a better join deciding on which constructor to use, please let us know.",
-                "Ticket 685 contains the discussion and a workaround for ambiguous constructors using inner class.",
-                "See https://github.com/mockito/mockito/issues/685"
+                join("", " - ", constructors)
```

Comment:
I originally took this message as a "workaround" until this PR is implemented, but you're right, let's keep it. Note there's a typo there ("join" instead of "job"). I'll fix that while I'm at it.